scripts/asm-installer/install_asm

```diff
@@ -2522,6 +2522,12 @@ configure_package() {
     kpt cfg set asm anthos.servicemesh.istiod-vs-name "istiod-vs-${REVISION_LABEL}"
   fi
 
+  if [[ "${CA}" == "mesh_ca" && "${USE_VM}" -eq 1 ]]; then
```

Comment:
Can we remove && "${USE_VM}"? The aliasing is needed when using mesh CA in general.

Reply:
Ok I think so

Reply:
Sure